packages/esm-admin-openconceptlab-app/src/subscription/subscription.component.tsx

```diff
@@ -36,11 +61,18 @@ const Subscription: React.FC = () => {
       const abortController = new AbortController();
 
       const response = await updateSubscription(
-        { ...subscription, url: subscriptionUrl, token: token },
+        {
+          ...subscription,
+          url: subscriptionUrl,
+          token: token,
+          validationType: validationType,
+          subscribedToSnapshot: isSubscribedToSnapshot,
+        },
         abortController,
       );
 
       if (response.ok) {
+        mutate('/ws/rest/v1/openconceptlab/subscription?v=full');
```

Comment:
So I think the proper way to do things here would be to do something like:

```ts
const updatedSubscription = {
	...subscription,
  url: subscriptionUrl,
  token: token,
  validationType: validationType,
  subscribedToSnapshot: isSubscribedToSnapshot,
};

mutate("/ws/rest/v1/openconceptlab/subscription?v=full", updatedSubscription, false);

const response = await updateSubscription(updatedSubscription);

mutate("/ws/rest/v1/openconceptlab/subscription?v=full");

if (response.ok) {
   // as before
}
```

Reply:
Fixed with de2ccd3